package-scripts/datadog-agent/preinst

```diff
@@ -46,10 +46,27 @@ if [ "$DISTRIBUTION" != "Darwin" ]; then
       useradd -r -M -g dd-agent -d $INSTALL_DIR -s /bin/sh \
         -c "Datadog Agent" dd-agent
     # Stop the old agent before installing
-    if [ -f "/etc/init.d/datadog-agent" ]; then
+    if [ -f "/etc/" ]; then
       /etc/init.d/datadog-agent stop || true
     fi
 
+    # Updates the yum repo to add the new GPG key for rhel based installations
```

Comment:
We could maybe add the same disclaimer as Debian: https://github.com/DataDog/dd-agent-omnibus/blob/40a28d59096d1ba606494ad580ced856f6e2ad35/package-scripts/datadog-agent/preinst#L35-L38